erpnext/projects/doctype/project/project.json

```diff
@@ -518,7 +526,6 @@
    "write": 1
   }
  ],
- "quick_entry": 1,
```

Comment:
enable quick_entry agian as it was there before.

Reply:
fixed